package.json

```diff
@@ -5,21 +5,23 @@
   "main": "index.js",
   "scripts": {
     "compile": "tsc",
-    "build": "rm -Rf dist && tsc -noEmit && ts-node ./build.ts",
+    "build": "rm -Rf dist && npm run lint && ts-node ./build.ts",
     "test": "jest",
-    "lint": "eslint src *.ts",
+    "lint": "tsc -noEmit && eslint src demo.ts",
```

Comment:
Any reason *.ts is removed? We have a build.ts outside src/ which would benefit from linting

Reply:
I took it out because it's _really_ far away from valid, my suggestion would be to get this one in order then do a follow up where we bring it back in